src/NServiceBus.Testing/ExpectedInvocations/ExpectHandleCurrentMessageLater.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace NServiceBus.Testing.ExpectedInvocations
+{
+    class ExpectHandleCurrentMessageLater : ExpectInvocation
+    {
+        public override void Validate(TestableMessageHandlerContext context)
+        {
+            if (!context.HandleCurrentMessageLaterWasCalled)
+            {
+                Fail(context.SentMessages);
```

Comment:
I'm not sure I follow. Are you talking about the exception message that is thrown?